adoc/chapters/programming_interface.adoc

```diff
@@ -2936,845 +2933,841 @@ Any errors from the <<command-group>> after it has been submitted are handled by
 passing <<async-error,asynchronous errors>> at specific times to an
 <<async-handler>>, as described in <<error-handling>>.
 
-A SYCL [code]#queue# can wait for all <<command-group,command groups>> that it
-has submitted by calling [code]#wait# or [code]#wait_and_throw#.
-
-The default constructor of the SYCL [code]#queue# class will construct a queue
```

Comment:
Why was this removed? because it was redundant with other part of the spec?

Reply:
Yes, exactly.  The description for each constructor contains this information.  No need to repeat it here.